### ISSUE TITLE
ENOENT: no such file or directory, open 'C:/Users/user.name/Projects/TechBright Services/techbright-crm/package.json'

### ISSUE THREAD

**Issue by techbright-limited** [VemtoOrg/vemto2-issues#64]
This happened upon creating a new project

**Error Message:**
ENOENT: no such file or directory, open 'C:/Users/user.name/Projects/TechBright Services/techbright-crm/package.json'

**Stack:**
```
Error: ENOENT: no such file or directory, open 'C:/Users/techbright-crm/Projects/TechBright Services/techbright-crm/package.json'
    at Object.openSync (node:fs:585:3)
    at Object.func [as openSync] (node:electron/js2c/asar_bundle:5:1812)
    at Object.readFileSync (node:fs:453:35)
    at e.readFileSync (node:electron/js2c/asar_bundle:5:9160)
    at $p.readFile (C:\Program Files\Vemto\resources\app.asar\build\src\main\main.js:1:5015)
    at C:\Program Files\Vemto\resources\app.asar\build\src\main\main.js:16:5792
    at C:\Program Files\Vemto\resources\app.asar\build\src\main\main.js:1:6782
    at Generator.next (<anonymous>)
    at C:\Program Files\Vemto\resources\app.asar\build\src\main\main.js:1:6611
    at new Promise (<anonymous>)
```

---

Vemto 2.0.6
win32 10.0.22631
Locale: en-GB
App version: 2.0.6

**Comment by TiagoSilvaPereira** [comment]
Hi @TechBright-Media I think the problem is happening due to the space in the folder name "TechBright Services". I'll fix that. For now you can try a folder without spaces

**Comment by ProfConsulting** [comment]
No it is not space problem, has the same issue:

ENOENT: no such file or directory, open 'C:/xampp/htdocs/vemto2/package.json'
Error: ENOENT: no such file or directory, open 'C:/xampp/htdocs/vemto2/package.json'

After creation I have only:
- vendor dir
- composer.json
- composer.lock

in the folder.

**Comment by TiagoSilvaPereira** [comment]
@ProfConsulting, thanks. We'll check what is happening.

**Comment by TiagoSilvaPereira** [comment]
@TechBright-Media @ProfConsulting please check https://github.com/TiagoSilvaPereira/vemto2-issues/issues/51

**Comment by ProfConsulting** [comment]
Wow! Upgraded php from 8.1 to 8.3 - working ... :)

**Comment by TiagoSilvaPereira** [comment]
@ProfConsulting awesome!!!

**Comment by TiagoSilvaPereira** [comment]
Fixed on Vemto 2.0.7
